invenio_accounts/ext.py

```diff
@@ -222,9 +222,12 @@ def init_config(self, app):
 
         :param app: The Flask application.
         """
-        if find_spec("celery") is not None:
-            app.config.setdefault("ACCOUNTS_USE_CELERY", not (app.debug or app.testing))
-        else:
+        try:
```

Comment:
this is not completely the same as before, because if find_spec("celery") does return None `app.config.setdefault("ACCOUNTS_USE_CELERY", False)` should also be called

Reply:
```
from contextlib import suppress

app.config.setdefault("ACCOUNTS_USE_CELERY", False)

with suppress(AttributeError):
    if find_spec("celery") is not None:
        app.config.setdefault("ACCOUNTS_USE_CELERY", not (app.debug or app.testing))
```